silverback/cluster/client.py

```diff
@@ -299,10 +300,10 @@ def bots(self) -> dict[str, Bot]:
     def bots_list(self) -> dict[str, list[Bot]]:
         response = self.get("/bots")
         handle_error_with_response(response)
-        bots_dict: dict[str, list[Bot]] = {}
+        bots_dict = defaultdict(list)
         for bot in map(Bot.model_validate, response.json()):
-            bots_dict.setdefault(bot.name, []).append(bot)
-        return bots_dict
+            bots_dict[bot.name].append(bot)
+        return dict(bots_dict)
```

Comment:
`default` dict should be a subclass of dict, so no need for the cast here

```suggestion
        return bots_dict
```